pkg/agent/agent.go

```diff
@@ -509,10 +510,14 @@ func (f *Flows) onInterfaceAdded(iface ifaces.Interface) {
 			Debug("interface does not match the allow/exclusion filters. Ignoring")
 		return
 	}
-	alog.WithField("interface", iface).Info("interface detected. Registering flow ebpfFetcher")
-	if err := f.ebpf.Register(iface); err != nil {
+	alog.WithField("interface", iface).Info("interface detected. trying to attach TCX hook")
+	if err := f.ebpf.AttachTCX(iface); err != nil {
 		alog.WithField("interface", iface).WithError(err).
-			Warn("can't register flow ebpfFetcher. Ignoring")
-		return
+			Warn("can't attach to TCx hook flow ebpfFetcher. fall back to use legacy TC hook")
```

Comment:
nit: I would use an Info log rather then Warn as TCx still won't be supported in many configs hence that's still a "normal" behaviour in most cases? (so it doesn't scare the users :-) )